test/exe/Main.hs

```diff
@@ -3022,7 +3050,7 @@ mkRange :: Int -> Int -> Int -> Int -> Range
 mkRange a b c d = Range (Position a b) (Position c d)
 
 run :: Session a -> IO a
-run s = withTempDir $ \dir -> runInDir dir s
+run s = run' (const s)
```

Comment:
This change looks unrelated. How come? Whats the justification?

Reply:
I spotted some duplicate code and took the chance to refactor it. Hlint? Human lint!